presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestRemoveUnreferencedScalarLateralNodes.java

```diff
@@ -28,6 +29,7 @@
     public void testRemoveUnreferencedInput()
     {
         tester().assertThat(new RemoveUnreferencedScalarLateralNodes())
+                .withBefore(new CardinalityTraitCalculationRuleSet().rules())
```

Comment:
`.whenRunAfter`. Or `.{if,when}RunWith` -- in Iterative, there is no "before" rules, only rule reactor.